contrib/pkg/broker/test_broker/controller/controller.go

```diff
@@ -318,6 +319,7 @@ func CreateController() controller.Controller {
 				Bindable:       true,
 				PlanUpdateable: true,
 			},
+			DeprovisionFailTimes: -1,
```

Comment:
that needs to be fixed in code, 

if you set `DeprovisionFailTimes` to 0 then it should not fail at all. In your case you will have one fail and only the second call will be successful.